python/pyspark/sql/dataframe.py

```diff
@@ -1304,7 +1307,10 @@ def limit(self, num: int) -> "DataFrame":
     def offset(self, num: int) -> "DataFrame":
         """Returns a new :class: `DataFrame` by skipping the first `n` rows.
 
-        .. versionadded:: 3.5.0
+        .. versionadded:: 3.4.0
```

Comment:
`offset` was first added to Python Client only in 3.4, and then added to vanilla one in 3.5